Makefile

```diff
@@ -10,49 +10,67 @@
 ##########################################################################
 
 MODULE_big = sqlite_fdw
-OBJS = connection.o option.o deparse.o sqlite_query.o sqlite_fdw.o sqlite_data_norm.o
+OBJS = connection.o option.o deparse.o sqlite_query.o sqlite_fdw.o sqlite_data_norm.o sqlite_gis.o
 
 EXTENSION = sqlite_fdw
 DATA = sqlite_fdw--1.0.sql sqlite_fdw--1.0--1.1.sql
 
-REGRESS = extra/sqlite_fdw_post extra/bitstring extra/bool extra/float4 extra/float8 extra/int4 extra/int8 extra/numeric extra/out_of_range extra/timestamp extra/uuid extra/join extra/limit extra/aggregates extra/prepare extra/select_having extra/select extra/insert extra/update extra/encodings sqlite_fdw type aggregate selectfunc 
+ifdef ENABLE_GIS
+	override CFLAGS := $(CFLAGS) -DSQLITE_FDW_GIS_ENABLE=1
```

Comment:
@mkgrgis There are some problems with this file.
**1. This line causes a big problem.**  
It overrides the CFLAGS of the whole system build. It makes the build command lost a lot of build arguments.  
This is the gcc build command in default mode:  
```
gcc -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Werror=vla -Wendif-labels -Wmissing-format-attribute -Wimplicit-fallthrough=3 -Wcast-function-type -Wshadow=compatible-local -Wformat-security -fno-strict-aliasing -fwrapv -fexcess-precision=standard -Wno-format-truncation -Wno-stringop-truncation -g -ggdb -g -O0 -fno-omit-frame-pointer -fPIC -fvisibility=hidden -I. -I. -I../../src/include  -D_GNU_SOURCE   -c -o connection.o connection.c
```
This is the gcc build command when ENABLE_GIS:  
```
gcc  -DSQLITE_FDW_GIS_ENABLE=1 -fPIC -fvisibility=hidden -I. -I. -I../../src/include  -D_GNU_SOURCE   -c -o connection.o connection.c
```
You can see that a lot of build arguments missing. It will affect much to the behavior.  
For example, because `-g -O0` arguments has been lost, I cannot debug sqlite_fdw.  
Please only assign value to `PG_CFLAGS`. It is an internal variable which will be appended to `CFLAGS` later. Setting that variable will not affect other arguments.  
```
override PG_CFLAGS += -DSQLITE_FDW_GIS_ENABLE
```
The build command after fixed:  
```
gcc -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Werror=vla -Wendif-labels -Wmissing-format-attribute -Wimplicit-fallthrough=3 -Wcast-function-type -Wshadow=compatible-local -Wformat-security -fno-strict-aliasing -fwrapv -fexcess-precision=standard -Wno-format-truncation -Wno-stringop-truncation -g -ggdb -g -O0 -fno-omit-frame-pointer -DSQLITE_FDW_GIS_ENABLE -fPIC -fvisibility=hidden -I. -I. -I../../src/include  -D_GNU_SOURCE   -c -o connection.o connection.c
```
**2. You only need to specify `-DSQLITE_FDW_GIS_ENABLE`, no need to specify `-DSQLITE_FDW_GIS_ENABLE=1`.**  
**3. The format of Makefile does not need tabs like this. This is format of some language such as C, C++ ...**
![image](https://github.com/user-attachments/assets/e5a70db7-d9f6-462e-acd4-c4b2c78f3e78)

Please follow Makefile format like this (You can see that the color changed).:
![image](https://github.com/user-attachments/assets/51a01b07-1b1e-4520-9847-e38fd7a72b00)

Reply:
Thanks, @son-phamngoc ! This is very good `Makefile` style. I am not familiar with this formal language, hence here was only my ugly draft. Thanks for researches and correcting! Please verify a new version of the file.

Reply:
@mkgrgis Thanks for your update. It is OK now. Please help me to resolve this comment.